pynbody/halo/__init__.py

```diff
@@ -56,12 +62,27 @@ def is_subhalo(self, otherhalo):
 
         return self._halo_catalogue.is_subhalo(self._halo_id, otherhalo._halo_id)
 
+    def physical_units(self, distance='kpc', velocity='km s^-1', mass='Msol', persistent=True, convert_parent=True):
+        if convert_parent:
```

Comment:
To make sure I understand this distinction:

* with `convert_parent=True`, all arrays in the parent convert, and each subhalo (including this one) will have `physical_units` called with `convert_parent=False`
* with `convert_parent=False`, only this one halo has its properties converted and arrays are ignored

I find the comment here a bit hard to parse — I guess you are drawing attention to why `_autoconvert_properties` is _not_ called in the first code path? Maybe just find a way to make this comment a bit clearer.

Reply:
I have added a note in the docstring above. Is now clearer?

Reply:
Yes I think that's about as clear as it can be.

However there is a typo in your documented default value for `convert_parent` (it says `None` but the actual default is `True`)